src/stories/SelectWallet.stories.ts

```diff
@@ -0,0 +1,56 @@
+import { html, TemplateResult } from 'lit';
+import '../web-components/SelectWallet/index.js';
+
+export default {
+    title: 'Components/SelectWallet',
+    component: 'select-wallet',
+    argTypes: {
+        getUserWallets: {
+            control: { type: 'array' },
+            description: 'A function that returns an array of wallet objects.',
+            defaultValue: () => [],
+        },
```

Comment:
Is there a reason why this has to be a function? Can we just pass in an array of wallets?